### ISSUE TITLE
Error "Canceled:Query Changed while running" upon changing time range

### ISSUE THREAD

**Issue by TheRoSS** [grafana/grafana#19565]
**What happened**:
Open any slow query graph. While graph data is loading, try to change time range. You get an error "Canceled:Query Changed while running" and data stops loading.

**What you expected to happen**:
Current request to backend with old time range should be cancelled and new request with updated time range should be launched

**How to reproduce it (as minimally and precisely as possible)**:
see above

**Anything else we need to know?**:
I've developed a month picker panel plugin. One of its options is to always load current month. This option always redefines time range and I always get this error when this option is on.
So my issue can be splitted into 2 parts:
1. The time range switching bug itself
2. A kind of temporary solution for me: is there any way to load one plugin before others? If I could run my plugin before any requests to datasources it would partially solved my problem until the main bug will be fixed

**Environment**:
- Grafana version: 6.3.5
- Data source type & version: Clickhouse, mysql, any
- OS Grafana is installed on: Ubuntu 18.04
- User OS & Browser:
- Grafana plugins:
- Others:

**Comment by hugohaggmark** [comment]
HI @TheRoSS I'm not able to repro `Canceled:Query Changed while running` error on the official Grafana 6.3.5 image. Looking in the network tab Grafana does not cancel the ongoing `mysql request` when Grafana changes the TimeRange so that might be a bug. I think all data sources may handle this differently.

This is the query I tried:
```sql
SELECT
  createdAt AS "time",
  datacenter AS metric,
  id,
  benchmark(1000000, 1+1) as bench
FROM grafana_metric
WHERE
  $__timeFilter(createdAt)
ORDER BY createdAt
```

**Comment by TheRoSS** [comment]
Yes. You're right. It seems to be a clickhouse only bug (vertamedia-clickhouse-datasource). I will investigate it further and write the results here.

But another question remains open. How to load my plugin before any other plugins? I dug into grafana code and hadn't found the way.

**Comment by hugohaggmark** [comment]
@TheRoSS we use GitHub for issues and feature requests. For questions we kindly request that you ask your question at the [community site](https://community.grafana.com/) or in our external [Slack   here](https://grafana.slack.com/)

All the best

**Comment by torkelo** [comment]
that should be fixed in 6.4 then as that code is very different in 6.4
